src/NewLabel/NewLabel.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+import {get} from '../constants'
+import {LabelSizeKeys, labelVariants} from './_newLabelStyleUtils'
+
+export type LabelColorOptions =
+  | 'default'
+  | 'primary'
+  | 'secondary'
+  | 'accent'
+  | 'success'
+  | 'attention'
+  | 'severe'
+  | 'danger'
+  | 'done'
+  | 'sponsors'
+
+interface Props {
+  color?: LabelColorOptions
```

Comment:
Help wanted: `color` is not a good name for this prop. Something like `variant` or `appearance` feels better, but I don't like either of those either.

Any suggestions for this prop name?